### ISSUE TITLE
TypeError: require.context is not a function

### ISSUE THREAD

**Issue by tatyshev** [vue-styleguidist/vue-styleguidist#14]
Got the error `TypeError: require.context is not a function` when attempt to parse component containing webpack's [`require.context`](https://webpack.js.org/guides/dependency-management/#require-context)

```html
<template>
  <p>Some component</p>
</template>

<script>
  const ctx = require.context('some/path', true, /.jpg$/);

  export default {
    name: 'example',
  };
</script>
```

Console output:
```
Error when parsing someComponent.vue: TypeError: require.context is not a function
```

webpack configs are taken from official `vui-cli`/`webpack` template (https://git.io/v7UkZ)

```javascript
// styleguide.config.js

const webpack = require('./build/webpack.base.conf');

module.exports = {
  webpackConfig: webpack,
};
```

vue-styleguidist v1.2.0
webpack 2.6.1

**Comment by rafaesc92** [comment]
Ok, I've published the fix under v1.2.1.
Please try it and let me know if it works.

**Comment by tatyshev** [comment]
Great, that works. Thank you.
